### PR TITLE
agda: ghc and audit fix

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

**Comment by DomT4** [comment]
What's being fixed here?

File: Library/Formula/agda.rb [hunk 1]
```diff
@@ -10,13 +10,19 @@ class Agda < Formula
...
+
   bottle do
     sha256 "b5dfadfde1b45ca3d70f2ac812e98a6d72caf1d840d4cb9c4eee95e81d74ff0b" => :yosemite
     sha256 "4513aecea2fbf5d1a682e04e4cccfd795d48836bcf1b16f79c24fa3bd6482854" => :mavericks
     sha256 "d0e17808ee896cdedc499b010b67961dc3db3c430c501faef16c4b4482238447" => :mountain_lion
   end
 
-  head "https://github.com/agda/agda.git", :branch => "master"
+  head do
+    resource "stdlib" do
```
Comment (dunn) [inline Library/Formula/agda.rb]: The head `url` should be placed in this block as well, to keep everything in one place.

**Comment by mikemcquaid** [comment]
> What's being fixed here?

Would like some clarification on this too, thanks.

**Comment by nijikon** [comment]
This just makes ```brew audit --strict``` pass without errors.

**Comment by dunn** [comment]
Things that take a long time to build, like agda, aren't really worth updating for style fixes, since it still has to run through the test bots.

File: Library/Formula/agda.rb [hunk 2]
```diff
@@ -30,12 +36,6 @@ class Agda < Formula
...
 
-  head do
-    resource "stdlib" do
-      url "https://github.com/agda/agda-stdlib.git", :branch => "master"
-    end
-  end
-
   resource "stdlib" do
     url "https://github.com/agda/agda-stdlib.git", :branch => "2.4.2.4"
   end
```
Comment (xu-cheng) [inline Library/Formula/agda.rb]: It seems this should be put inside stable do block otherwise it will overwrite the resource for head.

**Comment by nijikon** [comment]
It looks like in order for the test to pass, ghc needs to be installed. What do we want to do here?

**Comment by DomT4** [comment]
Change the test, probably. I'm relying on people who actually use the software here, but apparently GHC isn't runtime, or at least isn't expected to be runtime.

**Comment by mikemcquaid** [comment]
> Change the test, probably.

Yep :+1:

**Comment by DomT4** [comment]
Will take a PR for an updated test that doesn't consistently break.

**Pull request closed without merging**